usagov/vote-gov-drupal
# Feature/vote 1637 update login page

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[VOTE-1637](https://bixal-projects.atlassian.net/browse/VOTE-1673)

## Description (optional)
This PR will:
- Enable and configure the USAgov Login module which will prompt users to login via SSO.
- Enable and configure the Config Ignore module which will help preserve system configurations across environments without being affected by config sync.
- Add custom block which will be maintained within the database to config ignore to avoid accidental removal during deployments (drush cim).
- Add environment based configs for samlauth.authentication within settings file.


## Deployment and testing (required, if applicable)

### Testing steps
1. Got to /user
2. Confirm button and title correctly displays configured text per PR requirements
3. Click on 'Proceed to GSA SecureAuth login' button and confirm that SSO login is successful
